templates/not_found.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link type="text/css" rel="stylesheet" href="/dist/stylesheets/index.css" />
+    <title>Not found</title>
+  </head>
+
+  <body class="not-found">
```

Comment:
Never use `body` as the Block (B in BEM), instead prefer to create a component `div`:

```
<body>
  <div class="not-found">

  </div>
</body>
```

Also, it might be better to create a layout later on for these error pages 💭 

```
<HTML class="layout-error">
```

Reply:
Fixed in 4fedf6614da6e5887fe0be91dad360de42998134

Reply:
That fix does not cover all the requirements as the html element still does not have any class name.

Since these pages have a different layout, prefer to create a new layout as advised before:

```
<html class="layout-error">
```

Then create new base styles for all error pages. It can be reused for 401 errors when a user is not authenticated and 403 when permissions are not sufficient.

The content can be left simple:

```
<body class="not-found">
  <div class="error-details">
    <h2 class="error-details__title"> 404 </h2>
    ...
  </div>
</body>
```

Currently, there are many issues:
- `<main>` is slightly overkill as there is no other content on the page

![image](https://user-images.githubusercontent.com/696529/104903128-339e5f80-59b2-11eb-9293-0ce573859a6f.png)


- Never use HTML tags in the class naming so `main-not-found` violates [this convention](https://nimblehq.co/compass/development/code-conventions/css/#class-names)

Reply:
Thanks for clarify this.
Fixed in 1295119899b58fc9ccdb3353ff7e7bcecbaaed24